src/jackdaw/serdes.clj

```diff
@@ -0,0 +1,8 @@
+(ns jackdaw.serdes
+  "Implements basic SerDes (Serializer/Deserializer)."
```

Comment:
There are other basic serdes to add here right? Along with no so basic ones like `uuid`

Reply:
Most like. After this is merged, feel free to any you find are missing.